public/js/monitor.js

```diff
@@ -57,59 +57,78 @@ function isValidEmail(val) {
   return re.test(String(val).toLowerCase());
 }
 
-function addClass(selector, className){
+function addClass(selector, className) {
   const el = document.querySelector(selector);
   if(el){
     el.classList.add(className);
   }
 }
 
-function removeClass(selector, className){
+function removeClass(selector, className) {
   const el = document.querySelector(selector);
   if(el){
     el.classList.remove(className);
   }
 }
 
-function removeInvalidMessage(){
+function removeInvalidMessage() {
   removeClass("#invalid-email-message", "show");
   removeClass(".input-group","invalid");
 }
 
-function addInvalidMessage(){
+function addInvalidMessage() {
   addClass("#invalid-email-message", "show");
   addClass(".input-group", "invalid");
 }
 
 function enableBtnIfEmailValid(e) {
+  removeInvalidMessage();
   const emailBtn = document.getElementById("submit-email");
   if (isValidEmail(e.target.value)) {
       emailBtn.disabled = false;
       removeInvalidMessage();
   } else {
       emailBtn.disabled = true;
+      if(e.code === "Enter") {
+        addInvalidMessage();
+      }
   }
 }
 
 function enableBtnIfInputEmpty(e) {
   const emailBtn = document.getElementById("submit-email");
-  if(e.target.value === ""){
+  if(e.target.value === "") {
       emailBtn.disabled = false;
       removeInvalidMessage();
   } else {
       enableBtnIfEmailValid(e);
   }
 }
 
-function showFalseDoor(){
+function closeByEscape(e) {
+  if(e.code === "Enter" || e.code === "Escape") {
+    addClass("#false-door", "hidden");
+    window.removeEventListener("keydown", (e) => {
```

Comment:
This doesn't work - it tries to remove a NEW function (that happens to call closeByEscape within it). It should be `window.removeEventListener("keydown", closeByEscape)` but even that wouldn't work, because the event listener that was added was an anonymous function that calls closeByEscape within it. This combination of add/remove would work though:

```
window.addEventListener("keydown", closeByEscape);
window.removeEventListener("keydown", closeByEscape);
```